mdlp/_mdlp.pyx

```diff
@@ -1,16 +1,26 @@
 from __future__ import division
-from cpython.mem cimport PyMem_Malloc
-from cpython.mem cimport PyMem_Free
 import numpy as np
 cimport numpy as np
-from scipy.stats import entropy
+
+from cpython.mem cimport PyMem_Free
```

Comment:
Why did you change the import order?

Reply:
I placed the imports in alphabetical order. I don't think this is part of PEP8 explicitly but I believe is good style. In fact I meant to move the cython import above the block of 'from' imports as well to the 'import' block.

Reply:
Fair. Your'e right, it is good style to alphabetize imports. Thanks 👍